src/cc_catalog_airflow/dags/provider_api_scripts/common/storage/image.py

```diff
@@ -307,19 +333,35 @@ def _flush_buffer(self):
         if buffer_length > 0:
             logger.info(
                 'Writing {} lines from buffer to disk.'
-                .format(buffer_length)
+                  .format(buffer_length)
             )
             with open(self._OUTPUT_PATH, 'a') as f:
                 f.writelines(self._image_buffer)
                 self._image_buffer = []
                 logger.debug(
                     'Total Images Processed so far:  {}'
-                    .format(self._total_images)
+                      .format(self._total_images)
```

Comment:
This is the only formatting change I disagree with. Please revert it.

Reply:
I have reverted it but pycodestyle complains about these lines saying continuation line unaligned for hanging indent.